app/lib/services.rb

```diff
@@ -10,4 +11,8 @@ def self.content_store
       disable_cache: true,
     )
   end
+
+  def self.rummager
```

Comment:
I think once you rebase you won't need this as it's already been added for the Services work